xova/apps/xova/fixvis.py

```diff
@@ -200,12 +208,18 @@ def fixms(msname):
 
     with tbl(msname + "::FIELD", ack=False) as t:
         if not np.all(t.getcol("NUM_POLY") == 0):
-            raise RuntimeError("Does not support time-variable reference centres")
+            logger.critical("UVW recompute does not support"
+                            " time-variable reference centres."
+                            " Your dataset will contain averaged"
+                            " UVW coordinates!")
+            return
```

Comment:
make this not fall completely on its arse when some wacky ephemeris is specified. Not sure why but the tests have NUM_POLY>=1 in the PHASE_DIR column....... This is just to make the current tests pass

Reply:
OK